### PR TITLE
test: small tweaks to cri-o test process

### PR DESCRIPTION
**Pull request by haircommander**
(really I want to debug why we're seeing `Error: Duplicate test name(s) in file "/var/tmp/go/src/github.com/cri-o/cri-o/test/./ctr_seccomp.bats": test_ctr_seccomp_profiles_runtime-2fdefault` and also not failing because we don't have parallel installed)
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

File: contrib/cirrus/lib.sh [hunk 1]
```diff
@@ -232,6 +230,7 @@ install_testing_deps() {
     echo "Installing latest upstream version of BATS"
     ooe.sh git clone https://github.com/bats-core/bats-core.git /tmp/bats
     cd /tmp/bats
+	echo "Testing against bats commit: $(git rev-parse HEAD)"
```
Comment (TomSweeneyRedHat) [inline contrib/cirrus/lib.sh:233]: if you're keeping this long term, you might want to tweak the space/tab issue here.

File: contrib/cirrus/packer/fedora_setup.sh [hunk 2]
```diff
@@ -76,6 +76,7 @@ ooe.sh sudo dnf -y install \
     openssl \
     openssl-devel \
     ostree-devel \
+	parallel \
```
Comment (TomSweeneyRedHat) [inline contrib/cirrus/packer/fedora_setup.sh:79]: spacing/tab?

Comment (haircommander) [reply contrib/cirrus/packer/fedora_setup.sh:79]: :smiley:

File: contrib/cirrus/packer/fedora_setup.sh [hunk 3]
```diff
@@ -22,7 +22,7 @@ install_ooe
 echo "Updating packages"
 ooe.sh sudo dnf update -y
 
-ooe.sh sudo dnf -y module install cri-o:1.13
+ooe.sh sudo dnf -y module install cri-o:1.17
```
Comment (saschagrunert) [inline contrib/cirrus/packer/fedora_setup.sh]: Do we already want to use 1.18 :question:

Comment (haircommander) [reply contrib/cirrus/packer/fedora_setup.sh]: baby steps :laughing: this PR doesn't even work properly right now lol

**Pull request closed without merging**